mcrouter/tools/mcpiper/test/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the LICENSE file
+# in the root directory of this source tree.
+
+if(NOT BUILD_TESTS)
+  return()
+endif()
+
+add_executable(
+  mcpiper_tests "${CMAKE_SOURCE_DIR}/mcrouter/test/main.cpp"
+  # McPiperVisitorTest.cpp
```

Comment:
This test depends on `CarbonTestMessages` whose thrift spec doesn't compile. The thrift file is marked as generated so I'm not sure what's the way to edit that.